client/src/features/auth/loginPage.tsx

```diff
@@ -7,18 +7,40 @@ import {
   FormControlLabel,
   Checkbox,
   Link,
+  Alert,
 } from "@mui/material";
-import Grid from "@mui/material/Unstable_Grid2";
+import { useNavigate } from "react-router-dom";
+import { useAppDispatch, useAppSelector } from "../../app/hooks";
+import { loginUser } from "../users/usersSlice";
+import { useForm } from "react-hook-form";
+import { useEffect } from "react";
 
 export const LoginForm = () => {
+  const dispatch = useAppDispatch();
+  const {error, loading, userInfo} = useAppSelector((state) => state.users)
+
+  const { register, handleSubmit } = useForm()
+  const navigate = useNavigate();
+ 
+  useEffect(() => {
+    if (userInfo) {
+      navigate('/me')
```

Comment:
navigate routes should not have an hardcoded value but rely on constants, or functions